### PR TITLE
Add AWS CLI in Docker image

### PR DESCRIPTION
**Pull request by artemnikitin**
At the moment we will need it for uploading `all-in-one.yaml` to S3 bucket during release.
In the future we may need it to spin up EKS cluster.

### CODE REVIEW

File: build/ci/Dockerfile [hunk 1]
```diff
@@ -35,3 +35,8 @@ RUN curl -fsSLO https://download.docker.com/linux/static/stable/x86_64/docker-${
   && tar xzf docker-${DOCKER_VERSION}.tgz --strip 1 \
                  -C /usr/local/bin docker/docker \
   && rm docker-${DOCKER_VERSION}.tgz
+
+# Install AWS CLI
+RUN curl -O https://bootstrap.pypa.io/get-pip.py && \
```
Comment (nkvoll) [inline build/ci/Dockerfile]: Do we want to consider grabbing from the distro instead (as they will be signed and so on)? E.g
`apt-get update && apt-get install -y awscli && apt-get clean`

Comment (artemnikitin) [reply build/ci/Dockerfile]: AWS recommends this way https://docs.aws.amazon.com/cli/latest/userguide/install-linux.html

Comment (artemnikitin) [reply build/ci/Dockerfile]: But in our case it's probably isn't a big difference...

Comment (artemnikitin) [reply build/ci/Dockerfile]: Fixed

Comment (thbkrkr) [reply build/ci/Dockerfile]: Note: `apt` will not provide us the latest version.
(`apt => aws-cli/1.11.13` < `pip => aws-cli/1.16.174`)

Comment (artemnikitin) [reply build/ci/Dockerfile]: Yeah, I know. It's 1 year old, but I think that for such basic things as upload file to S3 it should be ok

**Pull request merged**